Find a functionally equivalent code:android.content.RestrictionEntry.getChoiceEntries:COMMENT
Method Modifier: public      
Comment:/**
 * Returns the list of strings, set earlier, that will be presented as choices to the user.
 * @return the list of choices presented to the user.
 */

@@ -1,3 +1,3 @@
 {
-    return choices;
+    return mChoiceEntries;
 }

